index.js

```diff
@@ -45,6 +45,12 @@ xapp.init(
   () => (config.GRAVITY_XAPP_TOKEN = xapp.token)
 )
 
+let schema = null
+mergedSchema().then(merged => {
+  schema = merged
+  app.listen(port, () => info(`Listening on ${port}`))
+})
```

Comment:
Maybe all of the `app` setup should just move inside this promise and then `schema` can just be a `const`.